learn/contributing/contributing_to_docs.md

```diff
@@ -6,88 +6,87 @@ This documentation website is hosted in a [public GitHub repository](https://git
 
-- Fixing a small error, such as a typo, broken link, or bad word choice.
-- Making a change that you are particularly qualified for (such as creating a guide for some software that you have mastered).
-- **Solving an [existing issue](https://github.com/meilisearch/documentation/issues)**!
+- **Solve an [existing issue](https://github.com/meilisearch/documentation/issues)**!
+- Fix a small error, such as a typo or broken link
+- Create or improve content about something you know very well—for example, a guide on how to integrate Meilisearch with a tool you have mastered
 
-Unless your PR is really small, we recommend you first [create an issue](#creating-your-first-issue) to confirm that it's worth your time and that nobody else is working on it already.
+In most cases, it is a good idea to [create an issue](#creating-your-first-issue) before making a PR. This allows you to coordinate with the documentation team and find the best way of addressing the problem you want to solve.
 
 #### Creating your first PR
 
-There are many ways to create pull requests, but we'll cover the simplest method for creating a PR on these docs. Feel free to try it on this page!
-
-All you need to continue is a [GitHub account](https://github.com).
-
-1. On the documentation page you'd like to edit, scroll to the bottom and click **"Edit this page"** on the left. This will take you to GitHub.
-2. If you're not already signed in, do so now. You may be prompted to create a [fork](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo)—if so, do it.
-3. Use GitHub's text editor to make the change you want.
-4. Scroll down to where it says `Propose changes`. In the first text box, write a short description (for example, "Fix typo"). In the second text box, write a longer description of your changes if necessary.
-5. Click `Propose changes` to continue. You should see a page that says `Comparing changes`. This screen is prompting you to create a pull request.
-6. Make sure the base repository is `meilisearch/documentation` and the base is `main`. Don't worry about the other two boxes—they automatically target the changes you just made.
-7. Below, you can see the changes you made compared to the main branch (what's currently published on docs.meilisearch.com). When you're ready, click `Create pull request`.
-8. **Congrats, you made your first PR!** Someone from the documentation team will review your pull request shortly. They may ask for changes, so keep an eye on your GitHub notifications.
-9. If everything looks good, your work will be merged into the `main` branch and become part of the official documentation site. You are now a Meilisearch Contributor! 🚀
+To create a PR you need a [GitHub account](https://github.com). Create one if necessary, then follow these steps:
+
+1. Go to the documentation page you'd like to edit, scroll down, and click **"Edit this page"** on the bottom left section of the page. This will take you to GitHub
```

Comment:
```suggestion
1. Go to the documentation page you'd like to edit, scroll down, and click "Edit this page"  at the bottom left of the screen. This will take you to GitHub
```
We use "page" a lot. Would it be a good idea to add a screenshot showing the "Edit this page" button?